pt/lessons/specifics/ecto.md

```diff
@@ -0,0 +1,368 @@
+---
+layout: page
+title: Ecto
+category: specifics
+order: 2
+lang: pt
+---
+
+Ecto é um projecto oficial do Elixir que fornece um invólucro de base de dados e
```

Comment:
I think would be better :

Ecto é um projeto oficial do Elixir que fornece uma camada de banco de dados e linguagem integrada para consultas. Com Ecto podemos criar *migrations*, definir modelos, inserir e atualizar registros, e fazer consultas.